release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3158,6 +3158,36 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-14-7"]
+=== RHSA-2023:7831 - {product-title} 4.14.7 bug fix and security update
+
+Issued: 2024-01-03
+
+{product-title} release 4.14.7, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:7831[RHSA-2023:7831] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:7834[RHBA-2023:7834] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.7 --pullspecs
+----
+
+[id="ocp-4-14-7-bug-fixes"]
+==== Bug fixes
+
+* Previously, when restarting ipsec pod, it killed existing policies. With this release, we also make sure to restart ipsec service, which will reinstate existing policies and solve the issue. (link:https://issues.redhat.com/browse/OCPBUGS-24633[*OCPBUGS-24633*])
+
+* Previously, updating a control plane machine set custom resource to reference an invalid resource, such as an invalid network name or image, created a control plane machine that would become stuck in the provisioning state and could not be deleted. With this release, this has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-23202[*OCPBUGS-23202*])
+
+* Previously, the Tuned profile reported `Degraded` condition after applying a PerformanceProfile due to trying to set a sysctl value a second time. With this release, the sysctl value is no longer set by tuned, instead it is only set by the `sysctl.d` file. (link:https://issues.redhat.com/browse/OCPBUGS-25305[*OCPBUGS-25305*])
```

Comment:
is it Tuned or TuneD? the [node tuning operator docs](https://docs.openshift.com/container-platform/4.14/scalability_and_performance/using-node-tuning-operator.html#custom-tuning-specification_node-tuning-operator) use TuneD. also a restructuring suggestion:
```suggestion
* Previously, applying a Performance Profile caused the TuneD profile to report a `Degraded` condition. This was because the generated TuneD profile was trying to set a second sysctl value. With this release, the sysctl value is no longer set by TuneD, instead it is only set by the `sysctl.d` file. (link:https://issues.redhat.com/browse/OCPBUGS-25305[*OCPBUGS-25305*])
```

Reply:
It seems to use both Tuned and TuneD in different sections which is confusing.